src/SingleHacker/SingleHackerLink.tsx

```diff
@@ -3,17 +3,24 @@ import * as React from 'react';
 import { Box } from '@rebass/grid';
 
 interface ILinkProps {
-  link: string | undefined;
+  link?: string;
+  linkText?: string;
   label: string;
+  newTab?: boolean;
 }
 
-const SingleHackerLink: React.SFC<ILinkProps> = ({ link, label }) => {
+const SingleHackerLink: React.SFC<ILinkProps> = ({
+  link,
+  linkText,
+  label,
+  newTab,
+}) => {
   return (
     <Box width={[1, 1 / 2]}>
       <strong>{label}</strong>:{' '}
       {link ? (
-        <a href={link} target="_blank">
-          {link}
+        <a href={link} target={newTab ? '_blank' : ''}>
```

Comment:
Do we need this level of granularity for the `newTab` prop? I would rather just assume that we want to open anything that's a URL in a new tab, and others (e.g.`mailto:`, `tel:`) in the same one.

Reply:
Do we use a regexp for http:// and https:// ?

Reply:
We can try to leverage this API? https://developer.mozilla.org/en-US/docs/Web/API/URL

We can check if it constructs and if its protocol is `http://` or `https://`

<img width="330" alt="image" src="https://user-images.githubusercontent.com/16010076/51957722-cdae1b80-241a-11e9-93a3-50621b83f8db.png">

Reply:
Done